tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -39,8 +39,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import java.util.concurrent.*;
```

Comment:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports
Disable wildcard imports